plugin-packs/postcss-preset-env/test/basic.css

```diff
@@ -425,3 +425,7 @@
 		content: ":scope in @scope, but with nesting";
 	}
 }
+
+@media (min-aspect-ratio: 1.77) {
+	/* media query aspect ratio : <number> */
+}
```

Comment:
This won't transform into anything before adding the plugin to `cssdb` and publishing a new version there.

Reply:
Updated `cssdb` and this now correctly transforms 🎉